### ISSUE TITLE
[feature] Add version number

### ISSUE THREAD

**Issue by Rowan-Paul** [Genymobile/scrcpy#626]
**Suggestion:**
Show the version number in the title bar (the title bar seems like the best option, but somewhere else could also work).

**Why?**
Currently there is no (easy) way to see the version number inside the app. Adding it in the title bar would be could be very handy to check if you're on the latest version.

**Comment by rom1v** [comment]
> Currently there is no (easy) way to see the version number inside the app.

Just for info, there is:

    scrcpy -v
    scrcpy --version

(but I guess that is what you referred to by "not easy")

**Comment by Rowan-Paul** [comment]
Yeah, I meant a visual way to see when using the GUI

**Comment by npes87184** [comment]
Or a splash screen which shows the scrcpy and its version?

**Comment by Rowan-Paul** [comment]
@npes87184  Good idea but I think the title would be better since it's visible all the time (though maybe it could be both)
